docs/writing-a-monitor.md

```diff
@@ -45,7 +45,7 @@ func init() {
 
 // Config for monitor
 type Config struct {
-	config.MonitorConfig `yaml:",inline" acceptsEndpoints:"true"`
+	config.MonitorConfig `yaml:",inline" acceptsEndpoints:"true" singleInstance:"true"`
```

Comment:
Don't make this part of this example, since monitors with discovery rules can't only be a single instance.